ui/panels-plugin/src/plugins/markdown/MarkdownPanel.tsx

```diff
@@ -25,10 +25,14 @@ function createMarkdownPanelStyles(theme: Theme) {
     // Make the content scrollable
     height: '100%',
     overflowY: 'auto',
-    // Ignore first margin
+    // Ignore top margin on the first element (TODO: Remove when we can set padding for each panel type)
     '& :first-child': {
       marginTop: 0,
     },
+    // Styles for headers
+    '& h1': {
+      fontSize: '2em',
```

Comment:
should this be using a value from `theme`?

Reply:
It looks like all the values in here are `em` based, so hopefully they're relative to a parent value set by the theme?